web/app.js

```diff
@@ -1005,7 +1005,10 @@ const PDFViewerApplication = {
     this._saveInProgress = false;
     this._hasAnnotationEditors = false;
 
-    promises.push(this.pdfScriptingManager.destroyPromise);
+    promises.push(
+      this.pdfScriptingManager.destroyPromise,
+      this.passwordPrompt.close()
```

Comment:
It doesn't seem to be related to what the title of the PR is about.

Reply:
That's why this was done in *separate* commit, since the change is small enough to not really warrant a separate PR in my opinion.